### PR TITLE
Create market on launch

### PR DESCRIPTION
**Pull request by mheguy**
Creates a market on init.

Checked for maps that called retailer and for maps that used the term 'market' and where appropriate, toggled the config option.

No more will people have to whine about me forgetting to plop a market on maps. :D

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -83,6 +83,8 @@ global.config = {
     -- adds a market
     market = {
         enabled = true,
+        -- will create a standard market on game startup
+        create_standard_market = true,
```
Comment (linaori) [inline config.lua:92]: What if we allow `false` or coordinates here instead? That means a scenario can decide where to put the market

Comment (mheguy) [reply config.lua:92]: A boolean or a table in a config option seems a touch confusing for a layperson tbh. How about I just add a field for coordinates?

**Review by grilledham — DISMISSED** [review]

**Comment by mheguy** [comment]
market.lua was badly conflicted and I could use another look at it again. I'll be able to test tomorrow.

**Review by linaori — APPROVED** [review]
Code wise the merge seems to be alright

**Review by grilledham — APPROVED** [review]

**Pull request merged**